data/data/aws/vpc/outputs.tf

```diff
@@ -18,14 +18,6 @@ output "worker_sg_id" {
   value = "${aws_security_group.worker.id}"
 }
 
-output "api_sg_id" {
-  value = "${aws_security_group.api.id}"
-}
-
-output "console_sg_id" {
```

Comment:
Final consumers for this and `api_sg_id` were removed in ec2cfa5021 (coreos/tectonic-installer#2177).  Nice catch :)